src/guides/v2.3/test/testing.md

```diff
@@ -87,6 +87,103 @@ They are executed during continuous integration using the `bin/magento` tool.
 
 Please see the [`magento dev:tests:run`][] documentation for more information, using the test type `static`.
 
+### PHPStan
+
+[PHPStan][] is a static analysis tool for PHP. Magento now integrates PHPStan by default.
```

Comment:
```suggestion
[PHPStan][] is a static analysis tool for PHP. Magento integrates PHPStan by default.
```

I know we use words like 'now' in release notes, but seems it could be weird in regular documentation. In a year, now is not the intended now? Just a thought.